requirements.txt

```diff
@@ -1,2 +1,3 @@
 sentence-transformers==2.0.0
-jina==2.0.0rc9
+jina[hub]==2.0
```

Comment:
check if needed

Reply:
Updated in https://github.com/jina-ai/executor-sentence-transformer/pull/3/commits/6f331d2e31b15f73dc955cacdf7161fe8c6d6f5d